README.md

```diff
@@ -262,7 +264,158 @@ export default NuxtAuthHandler({
 })
 ```
 
-Note that there's way more options inside the `nextAuth.options` object, see [here](https://next-auth.js.org/configuration/options#options) for all available options.
+##### Example with a custom Strapi JWT provider 
+
+Here's what a full config can look like, wee allow authentication via:
+- Strapi v4 JWTs with the `CredentialsProvider`
```

Comment:
this does not need to be a bullet list as there is only one bullet. Make it one sentence instead:
```suggestion
This section gives an example of how the `NuxtAuthHandler` can be configured to use Strapi JWTs for authentication via the `CredentialsProvider` provider.
```

Reply:
this is also unadressed @darwebdb